tui/keys_test.go

```diff
@@ -82,9 +82,13 @@ func TestLeftRightKeyWhileConfirm(t *testing.T) {
 	modal := tview.NewModal().SetText("Really?")
 	ui.pages.AddPage("confirm", modal, true, true)
 
-	key := ui.keyPressed(tcell.NewEventKey(tcell.KeyLeft, 'h', 0))
+	key := ui.keyPressed(tcell.NewEventKey(tcell.KeyLeft, 0, 0))
```

Comment:
I'm not super familiar with this, but from my testing, looks like `EventKey.Rune()` for left/right arrow key is 0.

For `h`/`l` key, it's 104/108 (with 256 or `tcell.KeyRune` as the `EventKey.Key()`).